bedrock/teach/templates/teach/smarton/security.html

```diff
@@ -0,0 +1,337 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "teach/smarton/base.html" %}
+
+{% block page_title %}{{ _('Get smart on cybersecurity') }}{% endblock %}
+{% block page_desc %}{{ _('Look both ways, stay updated, and make sure your passwords are as buff as you are. Get smart on cybersecurity.') }}{% endblock %}
+{% block page_image %}{{ static('img/teach/smarton/security/hero-cybersecurity-white-bg.png') }}{% endblock %}
+
+{% block body_id %}smarton-security{% endblock %}
+{% block body_class %}smarton theme-security{% endblock %}
+
+{# We use a localized hashtag for Spanish locales #}
+{% set hashtag = ' #SéUnExperto' if LANG.startswith('es-') else ' #SmartOn' %}
```

Comment:
Seeing as this is common on both pages I think it would make sense to move this to the base template, which then avoid needing to repeat it on each invidivual smarton page?